Tests/SwiftFormatTests/PrettyPrint/AttributeTests.swift

```diff
@@ -468,4 +468,103 @@ final class AttributeTests: PrettyPrintTestCase {
 
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 100)
   }
+
+  func testLineBreakBetweenAttributes() {
```

Comment:
Can you add a test that includes attributes surrounded by `#if`?

I would expect

```swift
@foo @foo2
#if FLAG
  @bar @baz
#endif
@quux func f() {}
```

to become

```swift
@foo
@foo2
#if FLAG
  @bar
  @baz
#endif
@quux
func f() {}
```

Reply:
yup